general/clients/installing-kodi.md

```diff
@@ -1,61 +1,101 @@
 ---
 uid: clients-kodi
-title: Kodi
+title: Installing Kodi Add-On
```

Comment:
Do we want have Installing in the title? There's more here than just installing it

Reply:
When it's installed it automatically launches into the configuration page.  And I was mostly basing this off what we did for the Mopidy page.

The URL is already "installing-kodi.html", so I was making that match. https://jellyfin.org/docs/general/clients/installing-kodi.html

Reply:
We could rename the file and the title to just Kodi

Reply:
If that's preferred.  We should also rename the Mopidy page as well in that case

Reply:
I agree that a general name makes more sense if we include more clients.